src/01-general/work-orders.apib

```diff
@@ -61,8 +61,8 @@ Update a draft work order.
 + Request (application/json;charset=utf-8)
 
     + Attributes (object)
-        + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string)
         + customer (object, required)
+        + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string, required)
```

Comment:
Don't mind this line moving, it's from editing a hunk manually. Fixed in next commit.